src/Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -314,12 +314,14 @@ This code manages the many-to-[one|many] association field popup
 
                         // reload the form element
                         jQuery('#field_widget_{{ id }}').closest('form').ajaxSubmit({
+                            // NEXT_MAJOR: Remove the 'code' parameter
                             url: '{{ path('sonata_admin_retrieve_form_element', {
```

Comment:
this one doesn't need `id`, `childId`, etc?

Reply:
Fixed